README.md

```diff
@@ -110,6 +110,7 @@ Output
   --format, -f                Format        [string] [choices: "json", "namespaced-json", "pot"] [default: "json"]
   --format-indentation, --fi  Format indentation (JSON/Namedspaced JSON)                  [string] [default: "\t"]
   --sort, -s                  Sort strings in alphabetical order                                         [boolean]
+  --sort-sensitivity, -ss     Sensitivity when sorting strings 										     [string]
```

Comment:
I think it would be good to add a dedicated section to the readme about sorting now. If I understand correctly, the `-ss` flag only has an effect in combination with the `-s` flag right? That should be noted.

Also this section can explain how strings will be sorted without the `-ss` flag, as well as what the different options are and what effect they have on the sort order.

Reply:
Limitations for the `-ss` flag have been added, as well as a separate section in the README